flink-runtime/src/main/java/org/apache/flink/runtime/dispatcher/Dispatcher.java

```diff
@@ -710,6 +710,12 @@ JobManagerRunner createJobManagerRunner(JobGraph jobGraph, long initializationTi
         return CompletableFuture.completedFuture(Acknowledge.get());
     }
 
+    @Override
+    public CompletableFuture<Acknowledge> shutDownClusterExceptionally(Throwable throwable) {
```

Comment:
Why do we want to shut down the cluster exceptionally? Wouldn't it be good enough in such a case to call the fatal exception handler and let it do its magic?

Reply:
removed